modular_skyrat/modules/gun_cargo/code/armament_datums/cantalan.dm

```diff
@@ -26,8 +26,9 @@
 
 /datum/armament_entry/cargo_gun/cantalan/pistol/ruby
```

Comment:
Honestly just remove this one, it looks bad

Reply:
Sure